### PR TITLE
Issue #19 render cell size depending on universe size

### PR DESCRIPTION
**Pull request by jprincipe**
converted from using `css` to `sass`
added classes for different universe sizes `sm`, `md`, `lg`, `xl` and `xxl`.
updated `universe_view` to figure out which class to apply to the universe based on dimensions

### CODE REVIEW

File: lib/game_of_life/universe/template.ex [hunk 1]
```diff
@@ -3,7 +3,7 @@ defmodule GameOfLife.Universe.Template do
   alias GameOfLife.Universe.Dimensions
   alias GameOfLife.Cell.Position
 
-  def dimensions("random"), do: %Dimensions{width: 16, height: 16}
+  def dimensions("random"), do: %Dimensions{width: 100, height: 100}
```
Comment (BGRicker) [inline lib/game_of_life/universe/template.ex:6]: 😍

File: lib/game_of_life_web/views/universe_view.ex [hunk 2]
```diff
@@ -7,17 +7,25 @@ defmodule GameOfLifeWeb.UniverseView do
...
-      content_tag(:div, class: "cell-row") do
-        Enum.map(0..(width - 1), fn x ->
-          generation
-          |> Generation.get_cell(%Position{x: x, y: y})
-          |> GameOfLifeWeb.CellView.render_cell()
-        end)
-      end
-    end)
+  def render_universe(%Universe{generation: generation, dimensions: %Dimensions{width: width, height: height} = dimensions}) do
+    content_tag(:section, class: universe_class(dimensions)) do
```
Comment (BGRicker) [inline lib/game_of_life_web/views/universe_view.ex:11]: ahhh this makes a lot of sense to set it up this way, pretty cool CSS class assignment helper

**Review by BGRicker — APPROVED** [review]

File: lib/game_of_life/universe/template.ex (hunk 1 above)
Comment (kevlarr) [reply lib/game_of_life/universe/template.ex:6]: 😻

File: assets/css/app.scss [hunk 3]
```diff
@@ -80,6 +80,31 @@ div[data-phx-view="GameOfLifeWeb.UniverseLive"] {
...
+
+    &.md .cell {
+        width: 10px;
+        height: 10px;
+    }
+
+    &.sm .cell {
+        width: 15px;
+        height: 15px;
+    }
```
Comment (kevlarr) [inline assets/css/app.scss:107]: Oh, cool!!!

File: lib/game_of_life_web/views/universe_view.ex [hunk 4]
```diff
@@ -7,17 +7,25 @@ defmodule GameOfLifeWeb.UniverseView do
...
+    end
   end
 
   def template_names(), do: Template.names()
+
+  defp universe_class(%Dimensions{width: width, height: height}) when width > 200 or height > 200, do: "universe xxl"
+  defp universe_class(%Dimensions{width: width, height: height}) when width > 125 or height > 125, do: "universe xl"
+  defp universe_class(%Dimensions{width: width, height: height}) when width > 75 or height > 75, do: "universe lg"
+  defp universe_class(%Dimensions{width: width, height: height}) when width < 15 or height < 15, do: "universe sm"
+  defp universe_class(_dimensions), do: "universe md"
```
Comment (kevlarr) [inline lib/game_of_life_web/views/universe_view.ex:30]: This is great!

**Review by kevlarr — APPROVED** [review]

**Pull request merged**